data/data.sh

```diff
@@ -3,5 +3,5 @@ set -e
 
 COMPOSE_PROJECT_NAME="landgriffon"
 
-docker compose -f ./docker-compose.yml --project-name ${COMPOSE_PROJECT_NAME} build
-docker compose -f ./docker-compose.yml --project-name ${COMPOSE_PROJECT_NAME} run --rm landgriffon-seed-data "$@"
+docker-compose -f ./docker-compose.yml --project-name ${COMPOSE_PROJECT_NAME} build
+docker-compose -f ./docker-compose.yml --project-name ${COMPOSE_PROJECT_NAME} run --rm landgriffon-seed-data "$@"
```

Comment:
we should update docker and force `docker compose` everywhere

Reply:
BE doesn't uses docker compose for anything so we should update our local machines to use `docker compose` since docker-compose is deprecated and no longer receives any updates: https://docs.docker.com/compose/migrate/
I think we should reject this change and use only `docker compose` command from now on.

Reply:
done!